src/components/EnvironmentOverride/ConfigMapOverrides.tsx

```diff
@@ -207,20 +214,40 @@ const OverrideConfigMapForm: React.FC<ConfigMapProps> = memo(function OverrideCo
             }
             return
         }
+        if (type === 'volume' && isFilePermissionChecked) {
+            if (!state.filePermission.value) {
+                dispatch({ type: 'filePermission', value: { value: state.filePermission.value, error: "Field is mandatory" } })
+                return
+            }
+            if (state.filePermission.value.startsWith("0")) { //Octal Format
```

Comment:
invert the logic.. if length is 4 then first character should be zero

Reply:
done